### PR TITLE
feat: show system message when user was removed from the team (AR-2477)

### PR DESCRIPTION
**Pull request by Garzas**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2477" title="AR-2477" target="_blank">StoryAR-2477</a>  Handle incremental events for teams
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

When user leaves or was removed from a team, then show system message in every conversation when he was a member without ranking those conversation up.

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

Screenshot 2022-10-12 at 10 12 08

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1091/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4659971)(4659971)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4659971)(4659971)-dev-release.apk)

**Review by yamilmedina — APPROVED** [review]

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -423,6 +423,7 @@
     </plurals>
     <string name="label_system_message_blocked_user">%1$s blocked this user</string>
     <string name="label_system_message_renamed_the_conversation">%1$s renamed the conversation\n%2$s</string>
+    <string name="label_system_message_team_member_left">%1$s was removed from the team</string>
```
Comment (ohassine) [inline app/src/main/res/values/strings.xml:427]: left or removed ? it two different actions
removed: removed by admin
left: leave the conversation by him self

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1091/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4660703)(4660703)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4660703)(4660703)-dev-release.apk)

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Garzas) [reply app/src/main/res/values/strings.xml:427]: We don't have information about it in event data

**Pull request merged**